src/app/component/pages/admin/admin-management/admin-management.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+
+import { AdminManagementComponent } from './admin-management.component';
+import {RouterTestingModule} from "@angular/router/testing";
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+
+
+describe('AdminManagementComponent', () => {
+  let component: AdminManagementComponent;
+  let fixture: ComponentFixture<AdminManagementComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [RouterTestingModule, HttpClientTestingModule],
+      declarations: [ AdminManagementComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AdminManagementComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Es fehlt noch ein Test für "onCreateSession()"

Reply:
Dient der Demo. Wird später in einem anderen Ticket noch umgesetzt.